sandbox/grommet-app/src/pages/layouts/kinds/OpsRamp/ResourceDetails.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Menu,
+  Text,
+  ResponsiveContext,
+  Layer,
+  Heading,
+} from 'grommet';
+import { LinkNext, StatusGoodSmall } from 'grommet-icons';
+import ContentPane from '../../../../components/ContentPane';
+import { ResourceAlerts } from './ResourceAlerts';
+import { ResourceTraits } from './ResourceTraits';
+import { ResourceOverview } from './ResourceOverview';
+
+interface ActionItem {
+  label: string | JSX.Element;
+  onClick: () => void;
+}
+
+const actionItems: ActionItem[] = [
+  { label: 'View details', onClick: () => {} },
+  { label: 'Copy ID', onClick: () => {} },
+  { label: 'Add relationships', onClick: () => {} },
+  { label: 'Assign credential', onClick: () => {} },
+  { label: 'Assign custom attribute', onClick: () => {} },
+  { label: 'Assign monitoring template', onClick: () => {} },
+  { label: 'Add maintenance window', onClick: () => {} },
+  { label: 'Add note', onClick: () => {} },
+  { label: 'Unmanage', onClick: () => {} },
+  {
+    label: <Text color="status-critical">Remove</Text>,
+    onClick: () => {},
+  },
+];
+
+interface ResourceHeaderProps {
+  name: string;
+  ipAddress: string;
+}
+
+const ResourceHeader: React.FC<ResourceHeaderProps> = ({ name, ipAddress }) => (
+  <Box pad={{ vertical: 'small' }} align="center" direction="row" gap="small">
+    <StatusGoodSmall color="status-ok" />
+    <Box>
+      <Text weight="bold">{name}</Text>
+      <Text size="small">{ipAddress}</Text>
+    </Box>
+  </Box>
+);
+
+interface ResourceActionsProps {
+  onClose: () => void;
+}
+
+const ResourceActions: React.FC<ResourceActionsProps> = ({ onClose }) => (
+  <Menu label="Actions" items={actionItems} onClick={onClose} />
```

Comment:
When I try to open this Menu, it actually just closes the detail panel.

Reply:
should be fixed